docs/content/en/docs/reference/clusterspec/cloudstack.md

```diff
@@ -5,4 +5,262 @@ weight: 30
+Any pods that you run on the control plane nodes must tolerate the taints you provide in the control plane configuration.
+>
+
+### controlPlaneConfiguration.labels
+A list of labels to apply to the control plane nodes of the cluster. This is in addition to the labels that
+EKS Anywhere will add by default.
+
+Modifying the labels associated with the control plane configuration will cause new nodes to be rolled out, replacing
+the existing nodes.
+
+### datacenterRef
+Refers to the Kubernetes object with CloudStack environment specific configuration. See `CloudStackDatacenterConfig Fields` below.
+
+### externalEtcdConfiguration.count
+Number of etcd members
+
+### externalEtcdConfiguration.machineGroupRef
+Refers to the Kubernetes object with CloudStack specific configuration for your etcd members. See `CloudStackMachineConfig Fields` below.
+
+### kubernetesVersion (required)
+The Kubernetes version you want to use for your cluster. Supported values: `1.23`, `1.22`, `1.21`, `1.20`
+
+### managementCluster
+
+### workerNodeGroupConfigurations (required)
+This takes in a list of node groups that you can define for your workers.
+You may define one or more worker node groups.
+
+### workerNodeGroupConfigurations.count (required)
+Number of worker nodes
+
+### workerNodeGroupConfigurations.machineGroupRef (required)
+Refers to the Kubernetes object with CloudStack specific configuration for your nodes. See `CloudStackMachineConfig Fields` below.
+
+### workerNodeGroupConfigurations.name (required)
+Name of the worker node group (default: md-0)
+
+### workerNodeGroupConfigurations.taints
+A list of taints to apply to the nodes in the worker node group.
+
+Modifying the taints associated with a worker node group configuration will cause new nodes to be rolled-out, replacing the existing nodes associated with the configuration.
+
+At least one node group must not have `NoSchedule` or `NoExecute` taints applied to it.
+
+### workerNodeGroupConfigurations.labels
+A list of labels to apply to the nodes in the worker node group. This is in addition to the labels that
+EKS Anywhere will add by default.
+
+Modifying the labels associated with a worker node group configuration will cause new nodes to be rolled out, replacing
+the existing nodes associated with the configuration.
+
+## CloudStackDatacenterConfig
+
+### availabilityZones.account (required)
+Account used to access CloudStack. The default is `admin`.
+
+### availabilityZones.credentialsRef (required)
+
+### availabilityZones.domain (required)
+CloudStack domain to deploy the cluster. The default is `ROOT`.
+
+### availabilityZones.managementApiEndpoint (required)
+Location of the CloudStack API management endpoint. For example, `http://10.11.0.2:8080/client/api`.
+
+### availabilityZones.name (required)
+Name of the CloudStack zone on which to deploy the cluster.
+
+### availabilityZones.zone.network (required)
```

Comment:
From the description, should this be rewritten as
```suggestion
### availabilityZones.zone.name (required)
```

Note that a user can also pass in `availabilityZones.zone.id` instead.

There should also be a separate line item for `availabilityZones.zone` which can either contain a name or an id

Reply:
I think I get what you are saying. Please check once I commit this.